app/_hub/kong-inc/oas-validation/overview/_index.md

```diff
@@ -23,6 +29,20 @@ To enable the plugin, use one of the following methods:
 
 {% endif_plugin_version %}
 
+{% if_plugin_version eq:3.7.x %}
```

Comment:
```suggestion
{% if_version gte:3.7.x %}
```
`gte` gives us "greater than or equal to", so the table will appear in subsequent versions too.